tests/components/prometheus/test_init.py

```diff
@@ -178,17 +403,19 @@ async def test_view_empty_namespace(
         "Objects collected during gc" in body
     )
 
-    assert (
-        'entity_available{domain="sensor",'
-        'entity="sensor.radio_energy",'
-        'friendly_name="Radio Energy"} 1.0' in body
-    )
+    EntityMetric(
+        metric_name="entity_available",
+        domain="sensor",
+        friendly_name="Radio Energy",
+        entity="sensor.radio_energy",
+    ).withValue(1.0).assertInBody(body)
```

Comment:
You don't need to change this, but I don't think you needed to append `.0` to all integer values.